### DISCUSSION TITLE
[灌水]有人认识吗？求个信息

### DISCUSSION THREAD

**Discussion by nninng** [share121/inter-knot#2859] [其他]
![IMG_20240818_225525](https://github.com/user-attachments/assets/6d38a8e3-72da-49af-a286-dd940ab9c37a)

今天晚上在报刊亭买报纸的时候，看到江边步道有一位狼希人，看起来好像在等谁的样子，因为有点小帅，所以偷拍了一张（手机镜头好糊），有人认识吗？希望有好心人给个信息（如果有侵权，我会删除这条帖子）。

**Comment by I-m-Miss-Lucy-s-dog** [comment]
貌似是某個家政所的執事吧 之前還看過某個小妹妹跟他所要尾巴毛 他還真的給了

**Comment by Kirolei** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >他叫灰太狼

**Comment by WORLD-executeME** [comment]
应该是维多利亚家政的执事莱卡恩先生

**Comment by Huanshenxuanlong** [comment]
维多利亚家政的人吧，他们办事效率好，就是服务太贵了
